src/families/bitcoin/logic.ts

```diff
@@ -143,8 +142,8 @@ type CoinLogic = {
   injectGetAddressParams?: (arg0: Account) => any;
 };
 
-const bchToCashaddrAddressWithoutPrefix = (recipient) =>
-  recipient ? bchaddr.toCashAddress(recipient).split(":")[1] : recipient;
+export const bchToCashaddrAddressWithoutPrefix = (recipient) =>
+  recipient ? recipient.substring(recipient.indexOf(":") + 1) : recipient;
```

Comment:
these are not equivalent. toCashAddress can sometimes also convert legacy addr.

```
toCashAddress('1B9UNtBfkkpgt8kVbwLN9ktE62QKnMbDzR') // bitcoincash:qph5kuz78czq00e3t85ugpgd7xmer5kr7c5f6jdpwk
```

and as we use bchToCashaddrAddressWithoutPrefix maybe for diff reasons, we will have to double check its impact. hopefully covered by test snapshot?